source/_posts/2015-10-09-the-slack-users-code.markdown

```diff
@@ -0,0 +1,258 @@
+---
+layout: post
+title: "The Slack Users' Code"
+date: 2015-10-09 22:30:00 -0500
+comments: true
+categories:
+---
+Without a code, [Slack](https://slack.com/) would be chaos. These are my ideas
+about what makes a good Slack team and Slack citizen. These are for _work Slack
+teams_. If you have a social Slack team set up, you do you because the
+expectations, goals and social forces are likely to be very different.
+
+
+## Yourself
+
+If you work in a widely distributed business unit, you should be interacting
+with people across the world from time to time. These are people who do not know
+you by sight. These four guidelines are to help those people know who you are
```

Comment:
“…to help those people get to know you…” maybe.

“help those people know who” is a strange construct and I tripped up on it, FWIW.